docs/core/tutorials/libraries.md

```diff
@@ -244,95 +198,49 @@ And in the middle of the source, you can use `#if` directives to use those libra
-            
-            int dotNetCount = Regex.Matches(result, ".NET").Count;
-            
-            return $"dotnetfoundation.orgmentions .NET {dotNetCount} times in its HTML!";
-        }
-#endif
-    }
-```
-
-When you build a project that includes `net40`, `net45`, and `netstandard1.6` as frameworks in the *project.json* file, it will have these subfolders in the */bin/debug* folder:
-
-```
-net40/
-net45/
-netstandard1.6/
-```
-
-### But What about Multitargeting with Portable Class Libraries?
+         // .NET 4.5+ can use async/await!
+         public async Task<string> GetDotNetCountAsync()
+         {
+             string url = "http://www.dotnetfoundation.org/";
 
-If you want to cross-compile with a PCL target, you must add a build definition in your `project.json` file under `buildOptions` in your PCL target.  You can then use `#if` directives in the source which use the build definition as a preprocessor symbol.
+             // HttpClient is thread-safe, so no need to explicitly lock here
+             var result = await _client.GetStringAsync(url);
 
-For example, if you want to target [PCL profile 328](http://embed.plnkr.co/03ck2dCtnJogBKHJ9EjY/preview) (The .NET Framework 4, Windows 8, Windows Phone Silverlight 8, Windows Phone 8.1, Silverlight 5), you could to refer to it to as "PORTABLE328" when cross-compiling.  Simply add it to the `project.json` file as a `buildOptions` attribute:
+             int dotNetCount = Regex.Matches(result, ".NET").Count;
 
-```json
-{
-    "frameworks":{
-        "netstandard1.6":{
-           "dependencies":{
-                "NETStandard.Library":"1.6.0",
-            }
-        },
-        ".NETPortable,Version=v4.0,Profile=Profile328":{
-            "buildOptions": {
-                "define": [ "PORTABLE328" ]
-            },
-            "frameworkAssemblies":{
-                "mscorlib":"",
-                "System":"",
-                "System.Core":"",
-                "System.Net"
-            }
-        }
+             return $"dotnetfoundation.orgmentions .NET {dotNetCount} times in its HTML!";
+         }
+ #endif
     }
 }
-
-```
-
-Now you can conditionally compile against that target:
-
-```csharp
-#if !PORTABLE328
-using System.Net.Http;
-using System.Threading.Tasks;
-// Potentially other namespaces which aren't compatible with Profile 328
-#endif
 ```
 
-Because `PORTABLE328` is now recognized by the compiler, the PCL Profile 328 library generated by a compiler will not include `System.Net.Http` or `System.Threading.Tasks`.
-
-When you build a project that includes PCL Profile 328 and `netstandard1.6` as frameworks in the *project.json* file, it will have these subfolders in the */bin/debug* folder:
+If you build this project with `dotnet build`, you'll notice three directories under the `bin/` folder:
 
 ```
-portable-net40+sl50+netcore45+wpa81+wp8/
-netstandard1.6/
+net40/
+net45/
+netstandard1.4/
 ```
 
+Each of these contain the `.dll` files for each target.
```

Comment:
Actually, we may not want this anymore.  I don't think `dotnet publish` is intended to be used for library development.